central/license/service/service_impl.go

```diff
@@ -3,53 +3,15 @@ package service
 import (
```

Comment:
Should we put a deprecation notice for these APIs so we can remove them eventually?

Reply:
Yes, will do this